src/maker/handlers.rs

```diff
@@ -254,6 +249,7 @@ impl Maker {
                 ContractSigsForSender { sigs },
             ))
         } else {
+            // Instead , we must create a MakerToTakerMessage variant stating about it rather than giving error back to Maker itself.
             Err(MakerError::General("not enough funds"))
```

Comment:
But that would require MakerError::General to take `String` not `&static str`.
IMO, we should use `string` because  it would help in writting better error messages , which we don't have for this error variant.

Reply:
just use a `&format!()`.